clients/ts/signalr/tests/Utils.ts

```diff
@@ -38,3 +38,26 @@ export class PromiseSource<T = void> implements Promise<T> {
         return this.promise.catch(onrejected);
     }
 }
+
+export class SyncPoint {
```

Comment:
Fancy!

Reply:
SyncPoint continues it's journey towards world domination.